docs/APDUSPEC.md

```diff
@@ -201,3 +212,51 @@ All other packets/chunks contain data chunks that are described below
 | ------- | --------- | ----------- | ------------------------ |
 | SIG     | byte (65) | Signature   |                          |
 | SW1-SW2 | byte (2)  | Return code | see list of return codes |
+
+
+---
+
+### INS_SIGN_MSG
+
+#### Command
+
+| Field   | Type        | Content                   | Expected   |
+| ------- | --------    | ------------------------- | ---------- |
+| CLA     | byte (1)    | Application Identifier    | 0x80       |
+| INS     | byte (1)    | Instruction ID            | 0x03       |
+| P1      | byte (1)    | Payload desc              | 0 = init   |
+|         |             |                           | 1 = add    |
+|         |             |                           | 2 = last   |
+| P2      | byte (1)    | ----                      | not used   |
+| L       | byte (1)    | Bytes in payload          | (depends)  |
+
+
+This instruction signs a message received in the payload
+
+The resulting sequence of chunks is as follows:
+
+##### First Packet
+
+| Field   | Type     | Content              | Expected |
+| ------- | -------- | -------------------- | -------- |
```

Comment:
This app allows a dynamic derivation path. Something similar to this: https://github.com/LedgerHQ/app-lisk/blob/master/docs/proto.md#get-public-keyaddress